### DISCUSSION TITLE
Is it possible to run ape testing without deploying any chain?

### DISCUSSION THREAD

**Discussion by LinuxIsCool** [ApeWorX/ape#1158] [Q&A] [answered]
I'm considering using ape framework as my default testing environment for software engineering of small little DeFi and TE data science libraries. 

If I don't need to execute any transactions in certain test suites, does Ape still spin up any chain infrastructure? Thanks! Shawn

**Comment by fubuloubu** [comment, marked as answer]
it should be the case that it will only launch a chain if you use it, but if that's the not the case and it messes with your library setup, then we can fix that bug

also, ape usually works with a local python-only network by default, so it probably won't affect anything at all if you ignore it
